### ISSUE TITLE
Add hint that this won't work in phonegap build.

### ISSUE THREAD

**Issue by istr** [mikaoelitiana/cordova-plugin-ios-no-export-compliance#1]
It could be worth noting in the README that this plugin fails to work in phonegap build.

Apart from that, are you sure that setting `CFBundleURLTypes` as the parent node really works?

I tried setting this using the method outlined in https://github.com/phonegap/build/issues/512 and http://stackoverflow.com/questions/35966608/adding-custom-parameter-phonegap-build/36496628. 

Trying this with a real app against itunes connect showed, however, that only setting the flag directly in the root (and not as a child entry of `CFBundleURLTypes`) had the desired effect.

I am pretty sure that your plugin needs to use

``` xml
<config-file platform="ios" target="*-Info.plist" parent="ITSAppUsesNonExemptEncryption">
  <false/>
</config-file>
```

instead of

``` xml
    <platform name="ios">
        <config-file target="*-Info.plist" parent="CFBundleURLTypes">
            <array>
                <dict>
                    <key>ITSAppUsesNonExemptEncryption</key>
                    <false/>
                </dict>
            </array>
        </config-file>
    </platform>
```

Note that I found out that omitting the explicit `platform="ios"` in the `config-file` tag fails at least for phonegap build: http://stackoverflow.com/questions/35798375/itsappusesnonexemptencryption-cordova-build/36498075#36498075.

Could you check/verify if this kind of nesting actually works in a plain cordova (not phonegap builder) environment?

**Comment by andres-torres-marroquin** [comment]
@lstr, it doesn't work on Cordova CLI, but after the changes you mentioned it actually worked.

**Comment by mikaoelitiana** [comment]
I have merged PR #2 so I think this can be closed now.
